usagov/usagov-2021
# Usagov 2069 tome slashes

## Jira Task


https://cm-jira.usa.gov/browse/USAGOV-2069

## Description

Expands the earlier fix for `/es/` to prevent Tome from exporting any local paths with a trailing slash that it finds in page content.

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Change Requirements



### Validation Steps

- Add links with trailing slash to content of a node: one for `http://localhost/es/` and for another page like `http://localhost/voting-and-elections`
- Export the site with tome
- Confirm `html/es/index.html` is not a redirect.
- Confirm `html/voting-and-elections/index.html` is not a redirect.
- Confirm count of redirects in new export is same versus an export from the `dev` branch

## Security Review




## Reviewer Reminders

- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions

Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
